src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -585,6 +585,7 @@ private void ProcessMTUSize(string targetNameOrAddress)
                     if (reply.Status == IPStatus.PacketTooBig || reply.Status == IPStatus.TimedOut)
                     {
                         HighMTUSize = CurrentMTUSize;
+                        replyResult = reply;
```

Comment:
looks like you now set `replyResult = reply` in each condition, so why not just have it set before the `if` statement?